src/Engine/Contest/ContestManager.cpp

```diff
@@ -39,7 +39,11 @@ ContestManager::ContestManager(const Contest _contest,
    dhv_xc_free(trace_full, true),
    dhv_xc_triangle(trace_triangle, predict_triangle, true),
    sis_at(trace_full),
-   net_coupe(trace_full)
+   net_coupe(trace_full),
+   weglide_free(),
```

Comment:
Just a note: this kind of explicit constructor call is redundant and useless.
The other constructor calls should be done in-line where the fields are declared, but the other (existing) fields aren't initialised that way either; that's a chance code code modernisation.